vcluster/reference/migrations/loft-cli-vcluster-cli-migration.mdx

```diff
@@ -0,0 +1,110 @@
+|-----------------------------------|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `loft start`                      | `vcluster platform start`                       |                                                                                                                                                                                                                                                                                                               |
+| `loft use management`             | `vcluster platform connect management`          |                                                                                                                                                                                                                                                                                                               |
+
+### misc commands
+
+| loft CLI                          | vcluster CLI                                    | Comments                                                                                                                                                                                                                                                                                                      |
+|-----------------------------------|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `loft backup`                     | `vcluster platform backup`                      |                                                                                                                                                                                                                                                                                                               |
+| `loft credits`                    | `vcluster credits`                              |                                                                                                                                                                                                                                                                                                               |
+
+### deleted commands
```

Comment:
Do we have comments on why these were removed?

Reply:
At the moment only for `generate admin-kube-config`. I'll find out for the others.

Reply:
I checked and it turns out that those others were more or less unused commands without purpose.